classes/mail/transport/PHPMailTransport.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/**
+ * @file classes/mail/transport/PHPMailTransport.php
+ *
+ * Copyright (c) 2014-2022 Simon Fraser University
+ * Copyright (c) 2000-2022 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PHPMailTransport.php
+ *
+ * @brief Transport/adapter to send with PHPMailer
+ */
+
+namespace PKP\mail\transport;
+
+use APP\core\Application;
+use PHPMailer\PHPMailer\PHPMailer;
+use PKP\config\Config;
+use PKP\core\PKPString;
+use Symfony\Component\Mailer\Envelope;
+use Symfony\Component\Mailer\SentMessage;
+use Symfony\Component\Mailer\Transport\TransportInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Mime\RawMessage;
+use Exception;
+use ReflectionObject;
+
+class PHPMailTransport implements TransportInterface
+{
+    public const MAIL_WRAP = 76;
+
+    /**
+     * @inheritDoc
+     */
+    public function send(RawMessage $symfonyMessage, Envelope $envelope = null): ?SentMessage
```

Comment:
I think it should be using `?Envelope`

Reply:
Ah, I just saw the `@inheritDoc`... If this is part of the interface, so be it 😁